client/landing/stepper/declarative-flow/internals/steps-repository/site-creation-step/index.tsx

```diff
@@ -1,16 +1,17 @@
 import { Site } from '@automattic/data-stores';
 import {
 	ECOMMERCE_FLOW,
+	StepContainer,
 	WOOEXPRESS_FLOW,
-	isLinkInBioFlow,
 	addPlanToCart,
+	addProductsToCart,
 	createSiteWithCart,
+	isCopySiteFlow,
 	isFreeFlow,
+	isLinkInBioFlow,
 	isMigrationFlow,
-	isCopySiteFlow,
+	isStartWritingFlow,
```

Comment:
I alphabetized these imports and the if statement for "Coming Soon" below. Sorry, I couldn't help myself. 😂  Otherwise there should be no change other than adding `isStartWritingFlow`.